corehq/apps/userreports/sql/connection.py

```diff
@@ -0,0 +1,65 @@
+import sqlalchemy
+from django.conf import settings
+
+
+DEFAULT_ENGINE_ID = 'default'
+
+
+def create_engine():
+    return sqlalchemy.create_engine(settings.SQL_REPORTING_DATABASE_URL)
+
+
+class ConnectionManager(object):
```

Comment:
thinking that we should reconcile this with https://github.com/dimagi/commcare-hq/blob/master/corehq/db.py#L7

@snopoke do you have thoughts on whether that sounds right? If so I can try and split the UCR-specific stuff from the general connection stuff and put the shared sessions through this class as well